src/braket/simulator_v2/base_simulator_v2.py

```diff
@@ -245,6 +249,14 @@ def _validate_jaqcd(self, circuit_ir, qubit_count: int, shots: int):
             )
 
 
+def _validate_thread():
```

Comment:
Can we add some kind of test for this warning?

Reply:
Done